demos/universal-sentence-encoder/README.md

```diff
@@ -53,7 +56,7 @@ Until it is published, the docker image with OpenVINO Model Server including the
 ```
 git clone -b develop https://github.com/openvinotoolkit/model_server
 cd model_server
-make docker_build SENTENCEPIECE=1 OV_SOURCE_ORG=rkazants OV_CONTRIB_ORG=rkazants OV_SOURCE_BRANCH=muse_perf OV_CONTRIB_BRANCH=rkazants/sentence_tokenizer_optimization OV_USE_BINARY=0
+make docker_build SENTENCEPIECE=1 OV_CONTRIB_ORG=rkazants OV_CONTRIB_BRANCH=rkazants/sentence_tokenizer_optimization OV_USE_BINARY=0
 
 ```
 
```

Comment:
1) in below command frozen model is still used + because building of ovms is required path to model will be incorrect
2) in client execution part there is no change dir required to execute 'pip install -r requirements.txt'